### ISSUE TITLE
Make all <b> tags <strong>

### ISSUE THREAD

**Issue by sharmi** [pythonindia/inpycon2019#65]
Both <b> tags and <strong> have the same css. So changing tags should be straightforward

**Comment by vikash212000yadav** [comment]
Hey guys, actually I'm new to this group, can someone please let me know what to do or how to access the files which requires any changes!

**Comment by sharmi** [comment]
Hi  Vikash,

      (My assumption is you are using linux)

         1. You have to clone the repo to be able to create pull requests
and contribute. This video gives a nice demo
https://www.youtube.com/watch?v=rgbCcBNZcdQ

         2. By now you will have a copy of the code in your local machine.
Follow the instructions in the  README.md to get website up and running
locally.

        3. Now you will need to modify the <b> tags in the index.md file to
fix the issue. Pl also check if there are other files where we are using
<b> tags and change appropriately.

        4. Continue following the instructions in the first video to create
a pull request. The website admin will do the merge.

If the above pointers are not clear or if you encounter any blockers,
please ask.

**Comment by sharmi** [comment]
@vikash212000yadav I saw now that you already contribute to several
projects.

Specifically for this bug, you need to modify the <b> tag in the index.md
file and _site directory.

If there is anything else specific, please let me know. Hope this helps.

**Comment by astronomersiva** [comment]
@vikash212000yadav, this site is built with [Jekyll](https://jekyllrb.com). You can check the project's README to get an overall idea on how to run your development server.

To fix this issue, you will have to replace all instances of `<b>` in the codebase with `<strong>`.

**Comment by astronomersiva** [comment]
@vikash212000yadav are you working on this?

**Comment by Rohithgilla12** [comment]
I want to take it up, if no one is doing it

**Comment by astronomersiva** [comment]
@Rohithgilla12 go ahead :+1:

**Comment by Rohithgilla12** [comment]
the _site directory is in gitignore, should i push changes to that also or leave it JLT
Thanks.
